README.md

```diff
@@ -132,11 +132,23 @@ You can create a dashboard with the NewRelic interface and export it to JSON for
 Then you can just copy it to a file within your entity type's folder, modify it if needed and refer to it from the definition.yml:
 
 ```yaml
-dashboardTemplates: 
- - ./<dashboardName>.json
+dashboardTemplates:
+ newRelic:
+  template: dashboard.json
+```
+
+If your entity has more than one telemetry source you can also provide a different dashboard for each of them. 
+The entity must have the tag `instrumentation.provider` with the value used as the key of the `dashboardTemplates`, in the example below we define a dashboard for two telemetry sources `newRelic` and `prometheus`.
```

Comment:
Maybe worth adding which ones should people use (which at the moment will be "newRelic" if using a NR agent, or "ask us" otherwise.  We are still pending a list of sources).

Reply:
That makes sense, I'll add that and change the example from `prometheus` to `otherSource` to make it generic/ 

Btw, do you think that it's ok to always require people to provide at least the newRelic one? (we're also having that discussion in the Emitter because if we don't have this as the default then we'd take the first one of the list regardless of tag).

Reply:
No, I think it's ok if NewRelic is also optional (e.g. you may have a type that is not supported by NR, but we can extract it from another integration).